index.html

```diff
@@ -35,7 +37,174 @@ <h1>I'm Safar Ali <br> A thriving <br> Software Developer </h1>
                 </ul>
             </div>
         </section>
-    </section>    
+    </section>
+
+    <!-- Main-section -->
+
+    <main class="main">
+
+        <!-- page-2 -->
+
+        <section class="page-2">
+            <div class="picture">
+                <a href="#"><img src="./images/picture-1.png" alt="picture-1"></a>
+            </div>
+            <div class="heading">
+                <h1>Tonic</h1>
```

Comment:
- [ ] It's crucial to follow semantics and best practices to improve our projects as much as we can! The `h1` is used as the main title of a website because represents the main heading/subject for the whole page so just used it once throughout your website😉 You could use from **h2 to h6** tags in a hierarchical manner with the rest of your texts 📝